db-demo-webtest/db-demo-product/pom.xml

```diff
@@ -1,7 +1,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.axonivy.market</groupId>
-  <artifactId>MY-PRODUCT-NAME-product</artifactId>
+  <artifactId>db-demo-product</artifactId>
```

Comment:
I think files under 'db-demo-webtest' added in this PR are duplicated and should not be part of this pr